README.md

```diff
@@ -79,17 +79,22 @@ You can set the index settings with this filter.
 
 By default, the searchable attributes is only the title.
```

Comment:
Wouldn't a better default include the title, content, excerpt and default taxonomies?